assets/maps/prefabs/space/prefab_MiniMart3.dmm

```diff
@@ -0,0 +1,325 @@
+	icon_state = "robust-broken";
+	name = "broken vending machine"
+	},
+/turf/unsimulated/floor/airless/twotone/black,
+/area/prefab/minimart)
+"D" = (
+/obj/cable/auto/purple/node,
+/obj/machinery/light/small/floor/cool/very,
+/obj/decal/cleanable/dirt,
+/obj/window/bulletproof/east,
+/obj/decal/cleanable/cobweb2,
+/obj/decal/cleanable/rust,
+/obj/machinery/vending/cigarette,
+/turf/unsimulated/floor/airless/twotone/black,
+/area/prefab/minimart)
+"F" = (
+/obj/cable/purple,
+/obj/cable/auto/purple/node,
+/obj/machinery/light/small/floor/cool/very,
+/obj/decal/cleanable/dirt,
+/obj/decal/cleanable/rust,
+/obj/machinery/vending/paint/broken,
+/turf/unsimulated/floor/airless/twotone/black,
+/area/prefab/minimart)
+"G" = (
+/obj/decal/cleanable/dirt,
+/turf/unsimulated/floor/airless/plating/damaged2,
+/area/prefab/minimart)
+"I" = (
+/obj/decal/cleanable/dirt,
+/obj/decal/cleanable/blood/tracks,
+/obj/decal/cleanable/sec_tape,
+/turf/unsimulated/floor/airless/plating/damaged2,
+/area/prefab/minimart)
+"J" = (
+/obj/machinery/light/runway_light/delay4,
+/obj/lattice/auto/turf_attaching,
+/turf/space,
+/area/prefab/minimart)
+"K" = (
+/obj/decal/cleanable/ripped_poster,
+/turf/unsimulated/wall/auto/adventure/shuttle/dark,
+/area/prefab/minimart)
+"L" = (
+/obj/decal/cleanable/dirt/dirt5,
+/turf/unsimulated/floor/airless/plating/damaged3,
+/area/prefab/minimart)
+"O" = (
+/obj/decal/cleanable/dirt/dirt2,
+/obj/decal/cleanable/blood/tracks,
+/turf/unsimulated/floor/airless/plating/damaged2,
+/area/prefab/minimart)
+"P" = (
+/obj/decal/cleanable/dirt/dirt2,
+/turf/unsimulated/floor/airless/plating/damaged3,
+/area/prefab/minimart)
+"Q" = (
+/obj/decal/tile_edge/floorguide/arrow_e{
+	pixel_x = 10
+	},
+/obj/decal/cleanable/gang_graffiti,
+/turf/simulated/floor/airless/caution/east,
+/area/prefab/minimart)
+"R" = (
+/obj/lattice/auto/turf_attaching,
+/turf/space,
+/area/prefab/minimart)
+"S" = (
+/obj/decal/tile_edge/floorguide/arrow_e{
+	pixel_x = 7
+	},
+/obj/fakeobject/rockpile,
+/obj/decal/cleanable/robot_debris/gib,
+/turf/simulated/floor/airless/caution/west,
+/area/prefab/minimart)
+"T" = (
+/obj/decal/tile_edge/floorguide/arrow_e{
+	pixel_x = 10
+	},
+/obj/decal/cleanable/wood_debris,
+/obj/decoration/damagedchair,
+/turf/simulated/floor/airless/caution/east,
+/area/prefab/minimart)
+"U" = (
+/obj/lattice/auto/turf_attaching,
+/obj/item/casing/small{
+	pixel_y = 5
+	},
+/obj/item/casing/small{
+	pixel_y = 3;
+	pixel_x = 3
+	},
+/obj/item/casing/small{
+	pixel_y = 3;
+	pixel_x = -3
+	},
+/obj/item/casing/small{
+	pixel_y = -3;
+	pixel_x = -3
+	},
+/obj/item/casing/small{
+	pixel_x = -5
+	},
+/turf/space,
+/area/prefab/minimart)
+"V" = (
+/obj/decal/cleanable/dirt/dirt2,
+/obj/decal/cleanable/sec_tape,
+/turf/unsimulated/floor/airless/plating/damaged1,
+/area/prefab/minimart)
+"W" = (
+/obj/cable/auto/purple/node,
+/obj/machinery/light/small/floor/cool/very,
+/obj/decal/cleanable/dirt,
+/obj/decal/cleanable/rust,
+/obj/item/gang_loot,
```

Comment:
this will message admins and give gangs points when opened. maybe make a new storage object with the same sprite (and make it clear it's not a real gang lootbag)?